### ISSUE TITLE
Restring 5.0.0 not working on API21

### ISSUE THREAD

**Issue by edgar-zigis** [B3nedikt/restring#46]
I just built your sample app for webview test on my API21 emulator and got this:

```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.content.res.Resources.updateConfiguration(android.content.res.Configuration, android.util.DisplayMetrics)' on a null object reference
        at dev.b3nedikt.restring.internal.RestringResources.updateConfiguration(RestringResources.kt:200)
        at android.content.res.Resources.<init>(Resources.java:232)
        at android.content.res.Resources.<init>(Resources.java:209)
        at dev.b3nedikt.restring.internal.RestringResources.<init>(RestringResources.kt:30)
        at dev.b3nedikt.restring.internal.RestringContextWrapper$res$2.invoke(RestringContextWrapper.kt:18)
        at dev.b3nedikt.restring.internal.RestringContextWrapper$res$2.invoke(RestringContextWrapper.kt:11)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at dev.b3nedikt.restring.internal.RestringContextWrapper.getRes(RestringContextWrapper.kt)
        at dev.b3nedikt.restring.internal.RestringContextWrapper.getResources(RestringContextWrapper.kt:21)
        at android.content.ContextWrapper.getResources(ContextWrapper.java:85)
        at androidx.appcompat.app.AppCompatDelegateImpl.attachBaseContext2(AppCompatDelegateImpl.java:420)
        at androidx.appcompat.app.AppCompatActivity.attachBaseContext(AppCompatActivity.java:107)
        at dev.b3nedikt.restring.example.BaseActivity.attachBaseContext(BaseActivity.kt:12)
        at android.app.Activity.attach(Activity.java:5876)
```

the app just fails to start.

**Comment by B3nedikt** [comment]
@edgar-zigis I fixed this in Restring 5.0.1.

**Comment by edgar-zigis** [comment]
Works 👍
